src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -331,7 +368,28 @@
 (def message-height-px 200)
 (def max-message-height-px 150)
 
-(defn on-long-press-fn [on-long-press message content]
+(defn message-pin-enabled [{:keys [public? group-chat from-community admins]}]
+  (let [current-pk @(re-frame/subscribe [:multiaccount/public-key])
```

Comment:
we already have this in the message
```
[message/chat-message
        (assoc message
               :incoming-group (and group-chat (not outgoing))
               :group-chat group-chat
               :public? public?
               :current-public-key current-public-key
               :show-input? show-input?)
               ```
so you can use this data, or if you need something else just add it there, we don't want to subscribe in every message

Reply:
you could assoc `:message-pin-enabled ` for every message

Reply:
Agree, forgot to update this file, just removed this function and moved the logic to the chat level. Also rebased to integrate with React Native Navigation changes.